rct229/rulesets/ashrae9012019/section22/section22rule35.py

```diff
@@ -0,0 +1,38 @@
+from rct229.rule_engine.rule_list_indexed_base import RuleDefinitionListIndexedBase
+from rct229.rule_engine.partial_rule_definition import PartialRuleDefinition
+from rct229.rule_engine.user_baseline_proposed_vals import UserBaselineProposedVals
+from rct229.rulesets.ashrae9012019.ruleset_functions.check_purchased_chw_hhw_status_dict import (
+    check_purchased_chw_hhw_status_dict,
+)
+
+
+class Section22Rule35(RuleDefinitionListIndexedBase):
+    """Rule 35 of ASHRAE 90.1-2019 Appendix G Section 22 (Chilled water loop)"""
+
+    def __init__(self):
+        super(Section22Rule35, self).__init__(
+            rmrs_used=UserBaselineProposedVals(False, True, False),
+            each_rule=Section22Rule35.RulesetModelInstanceRule(),
+            index_rmr="baseline",
+            id="22-35",
+            description="Baseline systems served by purchased chilled water shall not be modeled with"
+            " chilled water reset",
+            ruleset_section_title="HVAC - Chiller",
+            standard_section="Section G3.1.3.10 Chilled-Water Pumps (Systems 7, 8, 11, 12, and 13)",
+            is_primary_rule=False,
+            rmr_context="ruleset_model_instances/0",
```

Comment:
We typically don't use `rmr_context` in the `RuleDefinitionListIndexedBase` just because it is not used by this sub-class.
Alternatively, you can replace it with a `list_path="ruleset_model_instances[0]"`. It helps loop into the ruleset_model_instance in the nested class.